### PR TITLE
Provide a workflow for sending Dokka artifacts to gh-pages

### PR DESCRIPTION
**Pull request by CommanderTvis**

### CODE REVIEW

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ macOS-latest ]
```
Comment (altavir) [inline .github/workflows/pages.yml:12]: Why macos?

**Review by altavir — APPROVED** [review]

File: .github/workflows/pages.yml (hunk 1 above)
Comment (CommanderTvis) [reply .github/workflows/pages.yml:12]: Just copied build.yml, then removed Windows

Comment (CommanderTvis) [reply .github/workflows/pages.yml:12]: It does not matter of course

Comment (altavir) [reply .github/workflows/pages.yml:12]: You should probably use linux (it is easier to debug) and remove caches.

**Review by altavir — CHANGES_REQUESTED** [review]
Change to linux and remove konan caches

**Review by altavir — CHANGES_REQUESTED** [review]
Remove unnecessary OS matrix and caches

**Review by altavir — APPROVED** [review]

**Pull request merged**